src/components/molecules/NavigatorRowLabel/NavigatorRowLabel.tsx

```diff
@@ -350,6 +361,13 @@ const NavigatorRowLabel = (props: NavigatorRowLabelProps) => {
           </StyledIconsContainer>
         </Popover>
       )}
+      {isHovered && (!isInPreviewMode || (isInPreviewMode && previewType === 'cluster')) && (
```

Comment:
perhaps we should always show the menu and enable/disable menu actions depending on which state we're in? (since different actions might apply to different types of resources and/or previewModes)

Reply:
sure, I will do that in a following PR with either clone or rename resource